### ISSUE TITLE
Wrong link for troubleshotting

### ISSUE THREAD

**Issue by 4miners** [LiskHQ/lisk-scripts#108]
installLisk.sh
```
Removing Lisk directory and installation files

Please check installLisk.out for more details on the failure. See here for troubleshooting steps: https://docs.lisk.io/docs/core-troubleshooting

If no steps resolve your issue, please log an issue at: https://github.com/LiskHQ/lisk-build/issues
```

Link should be: `https://lisk.io/documentation/lisk-core/setup/troubleshooting`

Btw, would be nice to review all the links.

**Comment by Nazgolze** [comment]
Closed by #160

**Comment by Nazgolze** [comment]
Also just grepped all the scripts to make sure there was nothing else with docs.lisk.io :smile:
